modules/rostests/apitests/user32/ScrollBarRedraw.c

```diff
@@ -0,0 +1,727 @@
+            {
+                ok(Color == PrevColor,
+                   "CS_VREDRAW not specified, but vertical window expansion"
+                   " triggered unneccessary redraw, PrevColor: 0x%.8lX, Color: 0x%.8lX\n",
+                   PrevColor, Color);
+            }
+
+            SetWindowPos(Window, HWND_TOPMOST, 0, 0, SmallWidth, SmallHeight, SWP_NOMOVE);
+
+            FsmState = FSM_STATE_BOTH_SHRUNK;
+            break;
+
+        case FSM_STATE_BOTH_SHRUNK:
+            if (HaveHRedraw || HaveVRedraw)
+            {
+                ok(Color != PrevColor,
+                   "CS_HREDRAW or CS_VREDRAW specified, but combined"
+                   " vertical/horizontal shrinkage didn't trigger redraw,"
+                   " PrevColor: 0x%.8lX, Color: 0x%.8lX\n",
+                   PrevColor, Color);
+            }
+            else
+            {
+                ok(Color == PrevColor,
+                   "Neither CS_HREDRAW nor CS_VREDRAW specified, but combined"
+                   " vertical/horizontal shrinkage triggered unneccessary redraw,"
+                   " PrevColor: 0x%.8lX, Color: 0x%.8lX\n",
+                   PrevColor, Color);
+            }
+
+            SetWindowPos(Window, HWND_TOPMOST, 0, 0, OrigWidth, OrigHeight, SWP_NOMOVE);
+
+            FsmState = FSM_STATE_BOTH_EXPANDED;
+            break;
+
+        case FSM_STATE_BOTH_EXPANDED:
+            if (HaveHRedraw || HaveVRedraw)
+            {
+                ok(Color != PrevColor,
+                   "CS_HREDRAW or CS_VREDRAW specified, but combined"
+                   " vertical/horizontal expansion didn't trigger redraw,"
+                   " PrevColor: 0x%.8lX, Color: 0x%.8lX\n",
+                   PrevColor, Color);
+            }
+            else
+            {
+                ok(Color == PrevColor,
+                   "Neither CS_HREDRAW nor CS_VREDRAW specified, but combined"
+                   " vertical/horizontal expansion triggered unneccessary redraw,"
+                   " PrevColor: 0x%.8lX, Color: 0x%.8lX\n",
+                   PrevColor, Color);
+            }
+
+            FsmState = FSM_STATE_END;
+            DestroyWindow(Window);
+            break;
+
+        case FSM_STATE_END:
+            break;
+    }
+
+    PrevColor = Color;
+    return 0;
+}
+
+static int OnPaint(HWND Window)
+{
+    HRGN Region;
+    HDC hdc;
+    PAINTSTRUCT ps;
+
+    hdc = BeginPaint(Window, &ps);
+    if (hdc == NULL)
+    {
+        skip("Failed to get device context\n");
+        DestroyWindow(Window);
+        return 0;
+    }
+
+    Region = CreateRectRgn(ps.rcPaint.left,
+                           ps.rcPaint.top,
+                           ps.rcPaint.right,
+                           ps.rcPaint.bottom);
+    if (Region == NULL)
+    {
+        skip("Failed to create drawing region\n");
+        EndPaint(Window, &ps);
+        DestroyWindow(Window);
+        return 0;
+    }
+
+    if(!FillRgn(hdc, Region, ColorBrushes[CurrentColor]))
```

Comment:
```suggestion
    if (!FillRgn(hdc, Region, ColorBrushes[CurrentColor]))
```

Reply:
Thanks!  I missed this one, thanks for going ahead and committing the fix.